ql/src/test/org/apache/hadoop/hive/ql/exec/repl/TestRangerLoadTask.java

```diff
@@ -263,4 +267,56 @@ public void testSuccessDisableDenyRangerPolicies() throws Exception {
     //Deny policy is added
     Assert.assertEquals(1, actualPolicyList.getListSize());
   }
+
+  @Test
+  public void testRangerEndpointCreation() throws Exception {
+    URIBuilder uriBuilder = new URIBuilder("http://ranger.apache.org:6080");
```

Comment:
It tests actually URIBuilder, and not the testRanger endpoint creation flow. It would have been good if we could have had this test in RangerRestClient level. Otherwise the change looks to me.